dpnp/dparray.pyx

```diff
@@ -48,7 +48,7 @@ cimport dpnp.dpnp_utils as utils
 
 
 # initially copied from original
-cdef class _flagsobj:
+class _flagsobj:
```

Comment:
Do we really need changes related with dparray here? wouldn't it be better to move it to a separate PR, that could be merged before this one.